L1Trigger/L1THGCal/test/HGCalBackendStage1ParameterExtractor.cc

```diff
@@ -287,6 +221,35 @@ void HGCalBackendStage1ParameterExtractor::fillTriggerGeometry(json& json_file)
   }
 }
 
+std::tuple<uint32_t,uint32_t,uint32_t> HGCalBackendStage1ParameterExtractor::getTCaddress(int& tc_ueta, int& tc_vphi, bool& isScintillator) {
```

Comment:
- In principle, `tc_ueta` and `tc_vphi` are "in/out" arguments, they are modified within the function, so there is no need to return them in the tuple. One could only return the address.
- `isScintillator` is an input argument (not modified within the function), so should either be passed by value or by const reference